api/src/router/routes/provision.ts

```diff
@@ -21,11 +21,15 @@
 import { asyncMiddleware } from '@bcgov/common-nodejs-utils';
 import express from 'express';
 import { provisionCallbackHandler, provisionProfileNamespaces } from '../../controllers/provision';
+import { getAllProvisionedProfileIds, getProvisionedProfileBotJson } from '../../controllers/sync';
 
 const router = express.Router();
 
 // Provisioning
 router.post('/:profileId/namespace', asyncMiddleware(provisionProfileNamespaces));
 router.put('/namespace', asyncMiddleware(provisionCallbackHandler));
 
+// Bot-json-sync
+router.get('/sync/provisioned-profile-ids', asyncMiddleware(getAllProvisionedProfileIds));
+router.get('/sync/:profileId/provisioned-profile-bot-json', asyncMiddleware(getProvisionedProfileBotJson));
 export default router;
```

Comment:
Nit: no newline at end of file
```suggestion
export default router;

```